tests/sensu-agent.pp

```diff
@@ -1,3 +1,10 @@
+if $facts['os']['family'] == 'windows' {
+  $package_source = 'https://s3-us-west-2.amazonaws.com/sensu.io/sensu-go/5.7.0/sensu-go-agent_5.7.0.2380_en-US.x64.msi'
```

Comment:
is there a link somewhere for the latest release? this seems like it will be a pain to keep updated.

Reply:
I don't know, it's S3 so can't browse the parent directory in browser. Maybe we need to ask Sensu to provide a "latest" copy.  I'm not too concerned with this instance of the defined source always being latest but I would like more flexibility with the URL used for acceptance tests.  Having the URL be just version like `5.7.0` and not `5.7.0.2380` would at least allow us to pass the value as environment variable in appveyor config and not hardcode into tests.